ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/RecoveryManager.java

```diff
@@ -441,7 +442,8 @@ public static void main (String[] args)
                     }
                     catch(InterruptedException interruptedException)
                     {
-                        // do nothing
+                        System.err.println("The retry attempt was interrupted. Exiting...");
```

Comment:
Why no log message?

Reply:
It's because this printing is part of the `main` class which is invoked from the command line. I think it's more logical to print this message to STDERR rather than to log.